docs/source/transforms.rst

```diff
@@ -135,9 +141,9 @@ Vanilla Transforms
     :members:
     :special-members: __call__
 
-`RandUniformPatch`
+`RandCenterSpatialCrop`
 ~~~~~~~~~~~~~~~~~~~~
```

Comment:
this gives a warning 'WARNING: Title underline too short.'

Reply:
fixed in the latest patch.
Thanks.